### PR TITLE
Remove redundant patch and add comment explaining the patch

### PR DESCRIPTION
**Pull request by ocefpaf**
We don't need a new build number for this.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,9 @@ source:
   url: https://github.com/conda-forge/conda-smithy/releases/download/v{{ version }}/conda-smithy-{{ version }}.tar.gz
   sha256: 994d8a3e841879bda8637ffb170cbe8132f28623575817f1c8d66d54ef7a3eab
   patches:
-    - 1102.diff
+    # backport https://github.com/conda-forge/conda-smithy/pull/1102
+    # remove when that PR is merged.
+    - 1102.patch
```
Comment (scopatz) [inline recipe/meta.yaml]: It looks like you changed the name but didn't add `1102.patch`

Comment (ocefpaf) [reply recipe/meta.yaml]: It was already there. I preferred the `.patch` extension :smile:

Comment (scopatz) [reply recipe/meta.yaml]: weird!

Comment (isuruf) [reply recipe/meta.yaml]: Probably my fault. Thanks for fixing

**Pull request merged**